draft-ietf-cats-usecases-requirements.md

```diff
@@ -770,6 +770,134 @@ informative:
    WAN1 -> Cloud2 vCPE1 -> Cloud2 APP1
 
 
+##  Computing-Aware AI large model
+   
+   AI large model refers to a type of artificial intelligence model that 
+   is trained on massive amounts of data using deep learning techniques. 
+   These models are characterized by their large size, high complexity, 
+   and high computational requirements. AI large models have become 
+   increasingly important in various fields, such as natural language 
+   processing, computer vision, and speech recognition.
+   
+   There are usually two types of AI large models, AI foundation model 
+   and customized model. AI foundation large model is a model that can 
+   handle multiple tasks and domains, and has wider applicability and 
+   flexibility, but may not perform as well as customized model in 
+   specific domain tasks. Customized model is trained for specific 
+   industries or domains, and more focused on solving specific problems, 
+   but may not be applicable to other domains. AI foundation model 
+   usually involve mega-scale parametere, while customized model involves 
```

Comment:
What is a "mega-sclae parametere"?

Reply:
I saw the nit, but I don't see what is a "mega-scale parameter". I see what is a massive scale DC, data, etc. but not sure what these parameters are.

Reply:
From [ourworldindata](https://ourworldindata.org/grapher/artificial-intelligence-parameter-count#:~:text=Parameters%20are%20variables%20in%20an,in%20an%20artificial%20neural%20network.):
`Parameters are variables in an AI system whose values areadjusted during training to establish how input data get stransformed into the desired output.`

When training, the parameter is loaded into the GPU memory. So more parameters requires more GPUs to train it. 

Can we give a number for mega-scale parameter? For example, billion/trillion? Or how much resource it takes to train it.

Reply:
This [link](https://infohub.delltechnologies.com/l/generative-ai-in-the-enterprise/large-model-training/#:~:text=Large%20generative%20AI%20models%20have,single%20NVIDIA%20Tesla%20V100%20GPU.) gives an example.

Large generative AI models have significant compute requirements for training. According to OpenAI, for Chat GPT-3 with 175B parameters, the model size is approximately 350 GB, and it would take 355 years to train GPT-3 on a single NVIDIA Tesla V100 GPU.

So, is it ok to change to "AI foundation model usually involve billions of parameters"?